src/edge-functions/inject-script.ts

```diff
@@ -1,7 +1,7 @@
 // Documentation: https://sdk.netlify.com/docs
 
 import type { Config, Context } from "@netlify/edge-functions";
-import { HTMLRewriter } from "https://ghuc.cc/worker-tools/html-rewriter/index.ts";
+import { HTMLRewriter } from "https://deno.land/x/htmlrewriter/src/index.ts";
```

Comment:
How does this work? Shouldn't we include this in our codebase so that they can't update it without us knowing?

Reply:
this is the import they suggested in the review, it's [Netlify's implementation](https://github.com/netlify/htmlrewriter/tree/main), they didn't have versioning set up in the repo (i.e. `deno.json`).

but I [found it on deno.land](https://deno.land/x/htmlrewriter@v1.0.0) now, i can fix it to v1.0.0